$NetBSD: patch-src_lib__json_CMakeLists.txt,v 1.1 2019/04/09 07:08:18 wiz Exp $

We're not on Windows, so static libraries can have the same name as shared ones.

--- src/lib_json/CMakeLists.txt.orig	2017-12-20 21:07:10.000000000 +0000
+++ src/lib_json/CMakeLists.txt
@@ -96,7 +96,7 @@ IF(BUILD_STATIC_LIBS)
     ADD_LIBRARY(jsoncpp_lib_static STATIC ${PUBLIC_HEADERS} ${jsoncpp_sources})
     SET_TARGET_PROPERTIES( jsoncpp_lib_static PROPERTIES VERSION ${JSONCPP_VERSION} SOVERSION ${JSONCPP_SOVERSION})
     # avoid name clashes on windows as the shared import lib is also named jsoncpp.lib
-    if (NOT DEFINED STATIC_SUFFIX AND BUILD_SHARED_LIBS)
+    if (WINDOWS AND NOT DEFINED STATIC_SUFFIX AND BUILD_SHARED_LIBS)
       set (STATIC_SUFFIX "_static")
     endif ()
     set_target_properties (jsoncpp_lib_static PROPERTIES OUTPUT_NAME jsoncpp${STATIC_SUFFIX}
